.evergreen/auth_aws/aws_tester.py

```diff
@@ -201,6 +232,31 @@ def setup_web_identity():
     )
     create_user(ASSUMED_WEB_ROLE, kwargs)
 
+    return dict(AWS_WEB_IDENTITY_TOKEN_FILE=token_file, AWS_ROLE_ARN=role_arn)
+
+
+def handle_creds(creds: dict):
+    if "USER" in creds:
+        USER = quote_plus(creds["USER"])
+        PASS = quote_plus(creds["PASS"])
+        MONGODB_URI = f"mongodb://{USER}:{PASS}@localhost"
```

Comment:
```suggestion
        if "PASS" in creds:
            PASS = quote_plus(creds["PASS"])
            MONGODB_URI = f"mongodb://{USER}:{PASS}@localhost"
        else:
            MONGODB_URI = f"mongodb://{USER}@localhost"
```
To support a username without a password (an empty password vs. no password).